django/db/models/functions/datetime.py

```diff
@@ -243,9 +243,8 @@ class TruncDate(TruncBase):
     kind = 'date'
     lookup_name = 'date'
 
-    @cached_property
-    def output_field(self):
-        return DateField()
+    def __init__(self, *args, output_field=None, **kwargs):
```

Comment:
Is it better to exclude an explicit `output_field` kwarg so that it's not silently thrown away?